rpc/core/blocks.go

```diff
@@ -320,6 +325,10 @@ func Commit(heightPtr *int64) (*ctypes.ResultCommit, error) {
 // BlockResults gets ABCIResults at a given height.
 // If no height is provided, it will fetch results for the latest block.
 //
+// Results for the latest height are fetched from the state database, for
+// earlier heights - using the indexer. Note: indexer does not store EndBlock
+// ("end_block": null).
+//
```

Comment:
not sure what to do with EndBlock

Reply:
Can we index the EndBlock tags?

Reply:
Mmm.. We'll need a new event, but yes, certainly possible

Reply:
Wait.. why do we need tags in the endblock at all? 

we save params updates and validator updates separately and we also have events for them. so everyone can subscribe to them OR use RPC to fetch them at specific height.

Reply:
We may want to index certain events happening during an EndBlock - like if a validator gets slashed, and we want to query for all places where validator X got slashed